code/modules/food_and_drinks/recipes/tablecraft/recipes_misc.dm

```diff
@@ -275,7 +275,28 @@
 	name = "Onigiri"
 	reqs = list(
 		/obj/item/food/salad/boiledrice = 1,
-		/obj/item/food/seaweedsheet = 1,
+		/obj/item/food/seaweedsheet = 1
```

Comment:
```suggestion
		/obj/item/food/seaweedsheet = 1,
```
This list should still end with a comma 
![image](https://user-images.githubusercontent.com/66640614/186022695-b3d419f3-4335-481e-bc4a-17ae192ae212.png)